src/pages/frontpage/FrontPageHeading.tsx

```diff
@@ -0,0 +1,15 @@
+import { Box, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+export const FrontPageHeading = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1.5rem', mt: '3rem' }}>
+      <Typography variant="h1" sx={{ fontSize: '3rem', color: '#120732' }}>
```

Comment:
Denne kan endres til `primary.main`, eller er denne fargen noe som skal legges inn i theme etterhvert?
```suggestion
      <Typography variant="h1" sx={{ fontSize: '3rem', color: 'primary.main' }}>
```